openwrt/files/etc/rc.local

```diff
@@ -1,6 +1,9 @@
 # Put your custom commands here that should be executed once
 # the system init finished. By default this file does nothing.
 
+# Default config version to 0
+/root/bin/config-version.sh
```

Comment:
What should the new behavior be on boot? This was prior to the changes of:

`# If config ver is not passed then print out current version and exit
if [ "$CONFVER" == "" ]; then
   echo ${CONFLIST[0]} |  awk -F. '{print $NF}'
   exit 0
fi`

Reply:
Ah good catch, Im inclined to think we can just drop this all together. Im already creating the symlink as part of the build process: https://github.com/socallinuxexpo/scale-network/pull/203/files#diff-b0a351f9d8b5e7a78a6c384f198b231cR88